modules/apps/object/object-service/src/main/java/com/liferay/object/internal/related/models/ObjectEntry1to1ObjectRelatedModelsProviderImpl.java

```diff
@@ -147,21 +147,23 @@ public String getObjectRelationshipType() {
 
 	@Override
 	public List<ObjectEntry> getRelatedModels(
-			long groupId, long objectRelationshipId, long primaryKey, int start,
-			int end)
+			boolean bypassPermission, long groupId, long objectRelationshipId,
+			long primaryKey, int start, int end)
 		throws PortalException {
 
 		return _objectEntryLocalService.getOneToManyObjectEntries(
-			groupId, objectRelationshipId, primaryKey, true, 0, 1);
+			bypassPermission, groupId, objectRelationshipId, primaryKey, true,
+			0, 1);
 	}
 
 	@Override
 	public int getRelatedModelsCount(
-			long groupId, long objectRelationshipId, long primaryKey)
+			boolean bypassPermission, long groupId, long objectRelationshipId,
+			long primaryKey)
 		throws PortalException {
 
 		List<ObjectEntry> relatedModels = getRelatedModels(
-			groupId, objectRelationshipId, primaryKey, 0, 1);
+			false, groupId, objectRelationshipId, primaryKey, 0, 1);
```

Comment:
I know that we don't use 1-1 relationship, but why the getRelatedModelsCount() method is false and the getRelatedModels() is a variable that you send to _objectEntryLocalService.

Reply:
I missed this detail. Thanks for noticing it!